### PR TITLE
Created a new RC release version for version 0.11.0

### PR DESCRIPTION
**Pull request by andyi2it**
**What this PR does / why we need it**:
Creates a new pre-release for KServe

**Which issue(s) this PR fixes** :
https://github.com/kserve/kserve/issues/2676

**Type of changes**
Please delete options that are not relevant.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: python/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.10.1
\ No newline at end of file
+0.11.0-rc0
```
Comment (yuzisun) [inline python/VERSION]: The python version convention is 0.11.0rc0

File: charts/kserve-resources/values.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 kserve:
-  version: &defaultVersion v0.10.1
+  version: &defaultVersion v0.11.0-rc0
   modelmeshVersion: &defaultModelMeshVersion v0.10.0
```
Comment (ckadner) [inline charts/kserve-resources/values.yaml:3]: FYI: we will update the ModelMesh version here after the first KServe RC (@rafvasq)

Comment (ckadner) [reply charts/kserve-resources/values.yaml:3]: @rafvasq

**Pull request merged**